src/block_cache/block_chunkcache.c

```diff
@@ -916,6 +916,52 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
     return (0);
 }
 
+/*
+ * __wt_chunkcache_free_external --
+ *     Find chunks in the chunk cache using object id, and free the chunks.
+ */
+int
+__wt_chunkcache_free_external(
+  WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid, wt_off_t offset, uint32_t size)
+{
+    WT_CHUNKCACHE *chunkcache;
+    WT_CHUNKCACHE_CHUNK *chunk, *chunk_tmp;
+    WT_CHUNKCACHE_HASHID hash_id;
+    size_t already_read;
+    uint64_t bucket_id;
+    const char *object_name;
+    chunkcache = &S2C(session)->chunkcache;
+    already_read = 0;
+    object_name = NULL;
+
+    /* Only cache read-only tiered objects. */
```

Comment:
We should have a `WT_CHUNKCACHE_CONFIGURED` check, too.

Reply:
Is this necessary if we check this before we call the function?

Reply:
It wouldn't hurt then to assert that `WT_CHUNKCACHE_CONFIGURED` is set. That would document the expected usage for future developers as well as catching programming errors.

Reply:
added a check here [50bb019](https://github.com/wiredtiger/wiredtiger/pull/9838/commits/50bb01966791eb3b610cbf1f5dcc3a53a0a51162)